tests/components/nuheat/test_init.py

```diff
@@ -27,11 +27,12 @@ def tearDown(self):  # pylint: disable=invalid-name
     @patch("homeassistant.helpers.discovery.load_platform")
     def test_setup(self, mocked_nuheat, mocked_load):
         """Test setting up the NuHeat component."""
+        nuheat.nuheat = mocked_nuheat
```

Comment:
This is not good. Your mock will never be reverted and impact other tests. Use this instead:

```suggestion
        with patch.object(nuheat, 'nuheat', mocked_nuheat):
            …
```

Reply:
This was done in a couple of other PRs recently merged too. It would be good to fix those.

@djpremier probably knows which  integrations needed tests adjusting.

Reply:
Hi Martin, I did, as far as I can remember was more `melissa`, another one that needed` wake_on_lan`, but this one is with @fabaff, and I already commented on the correct form for it. But as soon as possible I analyze the tests and do the PR. Thanks.

Reply:
Yes, those are the ones. Thanks!